charts/opentelemetry-collector/values.yaml

```diff
@@ -72,10 +72,16 @@ presets:
   # Configures the Kubernetes Cluster Receiver to collect cluster-level metrics.
   # Adds the k8s_cluster receiver to the metrics pipeline
   # and adds the necessary rules to ClusterRole.
-  # Best used with mode = deployment or statefulset.
+  # Can be used with mode = deployment, statefulset, or daemonset.
+  # When running multiple replicas, the k8sleaderelector extension can be configured
+  # to ensure only one collector instance is collecting metrics at a time.
   # See https://opentelemetry.io/docs/kubernetes/collector/components/#kubernetes-cluster-receiver for details on the receiver.
   clusterMetrics:
     enabled: false
+    # Specify the k8sleaderelector extension ID to enable leader election when multiple replicas are used.
+    # This requires configuring and enabling the k8sleaderelector extension separately.
+    # Example: k8sleaderelector
+    k8sLeaderElector: ""
```

Comment:
We have, as a general rule, that presets should not require any user input to configure. Instead of a the preset accepting an extension id, could we do something like `enabledLeaderElection: false` and when enabled all the necessary configuration/components are automatically injected?

Reply:
Great suggestion! Implemented. The only thing I think is potentially a problem is that its now a "warning" when you install it without configuring the leader elector. If you want me to switch it to a "fail" I can.